servers/physics_3d/area_3d_sw.h

```diff
@@ -50,6 +50,10 @@ class Area3DSW : public CollisionObject3DSW {
 	real_t point_attenuation;
 	real_t linear_damp;
 	real_t angular_damp;
+	real_t wind_force_magnitude;
+	real_t wind_attenuation_factor;
```

Comment:
Missing initial values.

Reply:
this is what was causing the bug in the gui for me. good catch

Reply:
Is there a reason the other `real_t` parameters there don't have initial values?

Reply:
They are initialized in the constructor, which is how it was done before. We're changing the codebase to initialization in member declaration as a new coding style, but some parts of the code are still not updated.